src/functions/checks/checkGit.ts

```diff
@@ -10,7 +10,7 @@ import { CheckErrorSet } from "../../interfaces/types";
  */
 export const checkGit = async () : Promise<CheckErrorSet> => {
 
-  let returnSet : CheckErrorSet = {check : "git", error : false, details : null};
+  const returnSet : CheckErrorSet = {check : "git", error : false, details : null};
```

Comment:
Does this really work with const, it's attrubutes are edited just below this.

Reply:
yes